lib/potluck.rb

```diff
@@ -0,0 +1,49 @@
+require "pry"
+require "./lib/dish"
+
+class Potluck
+  attr_reader :date, :dishes
+  def initialize(date = "0-0-0")
+    @date = date
+    @dishes = []
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def get_all_from_category(category)
+    category_dishes = @dishes.map do |dish|
+      if dish.category == category
+        dish
+      end
+    end
+    category_dishes.compact
```

Comment:
👍 Think this is a perfectly good straightforward approach.

Take a look at `select` as a potential alternative as well!